contentctl/objects/deployment.py

```diff
@@ -36,8 +42,17 @@ def getDeployment(v:dict[str,Any], info:ValidationInfo)->Deployment:
             detection_name = info.data.get("name", None)
             if detection_name is None:
                 raise ValueError("Could not create inline deployment - Baseline or Detection lacking 'name' field,")
+
```

Comment:
We want to require certain fields to be defined in the YML files, but if one of these is created at runtime we will simply set them so that it passes validation.

Reply:
LGTM